<?xml version="1.0"?>

<launch>

  <param name="/use_sim_time" value="true"/>
  <arg name="grid_yaml_file"/>
  
  <!-- The grid_server node -->
  <node name="grid_server" pkg="ros_pam" type="grid_server.py" output="screen">
    <param name="yaml_file" value="$(arg grid_yaml_file)"/>
  </node>

</launch>

